sudoku/CS3243_P2_Sudoku_10.py

```diff
@@ -1,9 +1,12 @@
 import sys
 import copy
+import time
 
 # Running script: given code can be run with the command:
 # python file.py, ./path/to/init_state.txt ./output/output.txt
 
+start_time = time.time()
```

Comment:
Does it make a difference if the timing is started here instead of from line 269?

Reply:
Hmm I actually never thought about that. Let me try it.